internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -21,15 +21,16 @@ import (
 type kafkaStatus string
 
 const (
-	statusInstalling kafkaStatus = "installing"
-	statusReady      kafkaStatus = "ready"
-	statusError      kafkaStatus = "error"
-	statusRejected   kafkaStatus = "rejected"
-	statusDeleted    kafkaStatus = "deleted"
-	statusUnknown    kafkaStatus = "unknown"
-	strimziUpdating  string      = "StrimziUpdating"
-	kafkaUpdating    string      = "KafkaUpdating"
-	kafkaIBPUpdating string      = "KafkaIbpUpdating"
+	statusInstalling         kafkaStatus = "installing"
+	statusReady              kafkaStatus = "ready"
+	statusError              kafkaStatus = "error"
+	statusRejected           kafkaStatus = "rejected"
+	statusDeleted            kafkaStatus = "deleted"
+	statusUnknown            kafkaStatus = "unknown"
+	statusReassigningCluster kafkaStatus = "reassigningCluster"
```

Comment:
I am wondering if this should be something like:
- `rejectedClusterFull` 

What do you think?

Reply:
As in:
```golang
statusRejectedClusterFull kafkaStatus = "rejectedClusterFull"
```

I think you made a good point, it's a lot clearer.

Reply:
Yes.

Reply:
Perfect, thanks @machi1990. Those adjustments have been added.

Reply:
Thanks!